### DISCUSSION TITLE
following the tutorial.. unable to run index.js after creating DB Tables

### DISCUSSION THREAD

**Discussion by NewRay99** [prisma/prisma#13578] [Q&A]
after changing the index to include 

```
async function main() {
  await prisma.user.create({
    data: {
      name: 'Alice',
      email: 'alice@prisma.io',
      posts: {
        create: { title: 'Hello World' },
      },
      profile: {
        create: { bio: 'I like turtles' },
      },
    },
  })

  const allUsers = await prisma.user.findMany({
    include: {
      posts: true,
      profile: true,
    },
  })
  console.dir(allUsers, { depth: null })
}
```

unable to run index.js.

```
PrismaClientValidationError:
Invalid `prisma.user.create()` invocation in
C:\Users\....\source\repos\perm-man\msal-next-auth\pages\index.js:6:21

  3 const prisma = new PrismaClient()
  4
  5 async function main() {
→ 6   await prisma.user.create({
        data: {
          name: 'Alice',
          email: 'alice@prisma.io',
          posts: {
          ~~~~~
            create: {
              title: 'Hello World',
              updatedAt: 'Sun May 29 2022 16:53:17 GMT+0100 (British Summer Time)'
            }
          },
          profile: {
          ~~~~~~~
            create: {
              bio: 'I like turtles'
            }
          }
        }
      })

Unknown arg `posts` in data.posts for type UserCreateInput. Did you mean `Post`? Available args:
type UserCreateInput {
  email: String
```

https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/querying-the-database-typescript-sqlserver

**Comment by nurul3101** [comment]
Hey @NewRay99 👋 
Can you share your `schema.prisma` file?

**Reply by vladi-stevanovic** [reply]
👋 Hello @NewRay99  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
